sway-lsp/src/lib.rs

```diff
@@ -12,6 +12,8 @@ pub async fn start(config: DebugFlags) {
     let stdin = tokio::io::stdin();
     let stdout = tokio::io::stdout();
 
-    let (service, socket) = LspService::new(|client| Backend::new(client, config));
+    let (service, socket) = LspService::build(|client| Backend::new(client, config))
+        .custom_method("sway/runnables", Backend::runnables)
```

Comment:
In the future we should move these method name strings, param and response interfaces to a single file that has a corresponding file in the typescript client.